code/modules/jobs/job_types/security_officer.dm

```diff
@@ -278,3 +278,13 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 /obj/item/radio/headset/headset_sec/alt/department/sci
 	keyslot = new /obj/item/encryptionkey/headset_sec
 	keyslot2 = new /obj/item/encryptionkey/headset_sci
+
+	/datum/outfit/syndicate_empty/sbc/assault_komodo
```

Comment:
You accidentally nested this line underneath the previous block (starting with /obj/item/radio/headset/headset_sec/alt/department/sci). Delete a tab at the start of this line. You should also move this entire block up to before the "//Shiptest outfits end" comment in this file.

Reply:
Well, I meant to request changes, not approve, but still.

Reply:
That would potentially make the roles work?

Reply:
fixing the indentation absolutely will. Moving the code block up above the comment is just for the sake of organization.

Reply:
I'll have some time next sunday to get it fixed. If the bot marks this as stale somehow, im gonna hunt down its servers location and hurl insults at it

Reply:
this was fixed

Reply:
this was fixed